src/github_repo_request_local.py

```diff
@@ -101,27 +220,74 @@ def list_test_files(directory, excluded_extensions):
                 capture_output=True,
             )
             logger.info(f"Successfully cloned the repo: {repo_name}")
+
+            # Fetch the last commit hash and store it in the DataFrame
+            last_commit_hash = get_last_commit_hash(clone_dir)
+            df.at[index, "last_commit_hash"] = last_commit_hash
+
+            # On successful clone, increment the processed_count
+            processed_count += 1
+
         except subprocess.CalledProcessError as e:
             logger.error(
                 f"Failed to clone the repo: {repo_name}." f" " f"Exception: {e}"
             )
             df.at[index, "testfilecountlocal"] = 0
+
```

Comment:
I get the impression that 0 is not the best value to store for the count of test files; as 0 is also a valid result for projects (including this one) which don't currently have any test files. Using `None` or `-1` instead would make it clearer to someone reading the results that something went wrong when trying to obtain the information about the tests. 

At some point (possibly in a separate PR) we might want to consider how we could record the information about what happened and return it in the results (in turtle format). That way NLnet can review the results and understand what went wrong when we tried to gather the information about the tests for a particular project (where a project is a line of data in the original TSV file).

Reply:
- changed the value of `df.at[index, "testfilecountlocal"] ` to `-1`
- Created an issue [https://github.com/commercetest/nlnet/issues/49](url)